src/languages/es.js

```diff
@@ -303,6 +303,8 @@ export default {
         syncing: 'Sincronizando',
     },
     addSecondaryLoginPage: {
+        enterValidEmail: 'Please enter a valid email',
```

Comment:
@thesahindia, I have to add the translation (I will ask for it at slack )but you can start reviewing